doc-content/enterprise-only/admin-and-config/logging-events-database-proc.adoc

```diff
@@ -0,0 +1,78 @@
+[id='logging-events-database-proc']
+= Logging events to database
+
+You can log an event to a database that occurs on runtime in a process instance, node instance or variable instance.
+
+.Procedure
+. Map log classes to the data source that enables the data source to accept the log entries.
+* On {EAP}, edit the data source properties in the `persistence.xml` file.
+[source,code]
++
+----
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<persistence  version="1.0"  xsi:schemaLocation=
+    "http://java.sun.com/xml/ns/persistence
+     http://java.sun.com/xml/ns/persistence/persistence_1_0.xsd
+     http://java.sun.com/xml/ns/persistence/orm
+     http://java.sun.com/xml/ns/persistence/orm_1_0.xsd"
+  xmlns:orm="http://java.sun.com/xml/ns/persistence/orm"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://java.sun.com/xml/ns/persistence">
+
+  <persistence-unit name="org.jbpm.persistence.jpa">
+    <provider>org.hibernate.ejb.HibernatePersistence</provider>
+    <jta-data-source>jdbc/processInstanceDS</jta-data-source>
+    <class>org.drools.persistence.info.SessionInfo</class>
+    <class>org.jbpm.persistence.processinstance.ProcessInstanceInfo</class>
+    <class>org.drools.persistence.info.WorkItemInfo</class>
+    <class>org.jbpm.process.audit.ProcessInstanceLog</class>
+    <class>org.jbpm.process.audit.NodeInstanceLog</class>
+    <class>org.jbpm.process.audit.VariableInstanceLog</class>
+
+    <properties>
+      <property name="hibernate.dialect" value="org.hibernate.dialect.H2Dialect"/>
+      <property name="hibernate.max_fetch_depth" value="3"/>
+      <property name="hibernate.hbm2ddl.auto" value="update"/>
+      <property name="hibernate.show_sql" value="true"/>
+      <property name="hibernate.transaction.manager_lookup_class"
+                value="org.hibernate.transaction.BTMTransactionManagerLookup"/>
```

Comment:
@MarianMacik This looks like a deprecated configuration. IMHO the hibernate.transaction.jta.platform should be used here.

Reply:
+1